android/build.gradle

```diff
@@ -25,6 +25,9 @@ allprojects {
         mavenLocal()
         google()
         jcenter()
+        maven {
+            url "https://dl.bintray.com/textile/maven"
+        }
```

Comment:
This repo is necessary in order to resolve our `mobile` maven package (the aar).